lib/diplomat/kv.rb

```diff
@@ -42,6 +44,8 @@ def get key, options=nil, not_found=:reject, found=:return
       url += use_consistency(@options)
       url += dc(@options)
 
+      return_nil_values = (@options and @options[:nil_values])
```

Comment:
@PeterFCaswell I guess it's a matter of taste
would you like me to change to && ?